services/workspace_listing.py

```
@@ -0,0 +1,256 @@
+                        layouts = ctx.get("projectLayouts")
+                        if isinstance(layouts, list):
+                            if cid not in project_layouts_map:
+                                project_layouts_map[cid] = []
+                            for layout in layouts:
+                                if isinstance(layout, str):
+                                    try:
+                                        obj = json.loads(layout)
+                                        if isinstance(obj, dict) and obj.get("rootPath"):
+                                            project_layouts_map[cid].append(obj["rootPath"])
+                                    except Exception:
+                                        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)